script.js

```diff
@@ -0,0 +1,69 @@
+// create map
+
+//const { response } = require("express");
+
+// Create map:                                                       
+const myMap = L.map('map', {
+    center: [48.868672, 2.342130],
+    zoom: 12,
+});
+
+
+// Add OpenStreetMap tiles:
+L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+    minZoom: '15',
+}).addTo(myMap)
+
+
+// Get the user's coordinates:  
+let loc
```

Comment:
I don't think you need loc in the global scope -- you could define it in the function that uses it since that's the only place it's used :)